src/docs/sphinx/advancedExamples/validationStudies/thermalLeakyWell/Example.rst

```diff
@@ -0,0 +1,423 @@
+
+CO2 solubility in brine
+~~~~~~~~~~~~~~~~~~~~~~~
+
+As explained in :ref:`CO2-EOS`, we use the highly nonlinear model proposed by Duan and Sun (2004)
+to compute the CO2 solubility as a function of pressure, temperature, composition, and salinity.
+In `co2flash.txt`, we use the same parameters as above to construct the pressure-temperature tables
+of precomputed CO2 solubility in brine.
+
+.. code::
+
+        FlashModel CO2Solubility 6.8e6 2e7 5e4 302.0 307.5 0.25 1.901285269
+	
+------------------------------------------------------------------
+Initial and boundary conditions
+------------------------------------------------------------------
+
+The domain is initially saturated with brine with a hydrostatic pressure field and a geothermal
+gradient of 0.03 K/m.
+This is specified using the **HydrostaticEquilibrium** XML tag in the **FieldSpecifications** block:
+
+.. literalinclude:: ../../../../../../inputFiles/compositionalMultiphaseFlow/benchmarks/thermalLeakyWell/thermalLeakyWell_benchmark.xml
+    :language: xml
+    :start-after: <!-- SPHINX_HYDROSTATIC -->
+    :end-before: <!-- SPHINX_HYDROSTATIC_END -->
+
+Although this is the same block as in :ref:`ExampleIsothermalLeakyWell`, GEOSX is now enforcing the
+geothermal gradient specified in the **TableFunction** named ``initTempTable``, and is also accounting
+for the nonlinear temperature dependence of brine density to equilibrate the pressure field.
+
+We use the simple table-based approach shown below to impose the Dirichlet boundary conditions on the four
+sides of the domain.  
+
+.. literalinclude:: ../../../../../../inputFiles/compositionalMultiphaseFlow/benchmarks/thermalLeakyWell/thermalLeakyWell_benchmark.xml
+    :language: xml
+    :start-after: <!-- SPHINX_DIRICHLET_BC -->
+    :end-before: <!-- SPHINX_DIRICHLET_BC_END -->
+
+where the ``setNames = "{ east, west, south, north }"`` are defined using the **Box**
+XML tags of the **Geometry** section, and where the tables are defined as **TableFunction**
+in the **Functions** section. 
+		 
+.. note::
+   Due to the nonlinear dependence of brine density on temperature, this block does not exactly impose a Dirichlet pressure equal to the initial condition. Instead, here, we impose a linear pressure gradient along the z-axis, whose minimum and maximum values are the same as in the initial state. We could have imposed Dirichlet boundary conditions preserving the initial condition using as many points in `zlin.geos` as there are cells along the z-axis (instead of just two points).  
+
+The **SourceFlux** is the same as in the previous benchmark case (see :ref:`ExampleIsothermalLeakyWell`).
+
+---------------------------------
+Inspecting results
+---------------------------------
+
+We request VTK-format output files and use Paraview to visualize the results.
+The following figures show the distribution of CO2 saturation and pressure along the slice defined by x = 0 at t = 300 days.  
```

Comment:
200 or 300?  figure captions say 200 days.

Reply:
Yes, you are right, it should be 200 days